plover/config.py

```diff
@@ -383,7 +396,18 @@ def get_translation_frame_y(self):
         return self._get_int(TRANSLATION_FRAME_SECTION, 
                              TRANSLATION_FRAME_Y_OPTION,
                              DEFAULT_TRANSLATION_FRAME_Y)
-                             
+
+    def set_translation_frame_opacity(self, opacity):
```

Comment:
Makes sense for setting. I'll throw the assert in.

My thinking was along the lines "bogus opacity configuration would be a lame reason for a program someone relies on to do their job crashing", but if we're setting a bogus value, we should catch that during testing, and it's totally our fault.

For reading a value, that could be someone flubbing a manual edit to their config file, and I'd like to be forgiving of that.